Test/Tests/Component/NodeManager.Tests.ps1

```diff
@@ -123,20 +124,15 @@ Describe "Node manager" {
     BeforeEach {
         $Session = $MultiNode.Sessions[0]
 
-        Initialize-ComputeServices -Session $Session `
-            -SystemConfig $MultiNode.Configs.System `
-            -OpenStackConfig $MultiNode.Configs.OpenStack `
-            -ControllerConfig $MultiNode.Configs.Controller
         Clear-NodeMgrLogs -Session $Session
         Start-NodeMgr -Session $Session
     }
 
     AfterEach {
         try {
             Stop-NodeMgr -Session $Session
-            Clear-TestConfiguration -Session $Session -SystemConfig $MultiNode.Configs.System
         } finally {
-            Merge-Logs -LogSources (New-FileLogSource -Path (Get-ComputeLogsPath) -Sessions $Session)
+            Merge-Logs -DontCleanUp -LogSources (New-FileLogSource -Path (Get-ComputeLogsPath) -Sessions $Session)
```

Comment:
`-DontCleanUp` will cause subsequent test case log files to contain all the logs from previous test cases I think?

Reply:
All services would have to be stopped in order to gather logs without DontCleanUp flag and I'd argue that stopping every service after each test case isn't best practice. However, I think I'd like to change it and don't pass prepareenv flag to this tests altogether (i.e. components test) and limit this functionality to network tests only. What do you think?